docs/content/latest/yugabyte-cloud/cloud-monitor/cloud-queries-live.md

```diff
@@ -0,0 +1,30 @@
+---
+title: View live queries
+linkTitle: Live queries
+description: View live queries running on your cluster.
+headcontent:
+image: /images/section_icons/deploy/enterprise.png
+menu:
+  latest:
+    identifier: cloud-queries-live
+    parent: cloud-monitor
+    weight: 200
+isTocNested: true
+showAsideToc: true
+---
+
+Evaluate the performance of running queries on your cluster using the **Live Queries** on the cluster **Performance** tab. You can use this data to:
+
+- Visually identify relevant database operations.
```

Comment:
Should mention that live queries only show queries "in-flight" or currently in progress. This means queries that execute quickly might not show up on load of page.